packages/dicomImageLoader/src/imageLoader/wadors/metaData/metaDataProvider.ts

```diff
@@ -266,6 +267,28 @@ function metaDataProvider(type, imageId) {
       actualFrameDuration: getNumberValue(metaData['00181242']),
     };
   }
+
+  // This is used for gathering all the metadata for export
+  if (type === 'dataset') {
+    const types = [
+      'multiframeModule',
+      'generalSeriesModule',
+      'patientStudyModule',
+      'nmMultiframeGeometryModule',
+      'imagePlaneModule',
+      'imagePixelModule',
+      'voiLutModule',
+      'modalityLutModule',
+      'sopCommonModule',
+      'petIsotopeModule',
+      'overlayPlaneModule',
+      'transferSyntax',
+      'petSeriesModule',
+      'petImageModule',
+    ];
+
+    return getDatasetModule(imageId, metaDataProvider, types);
```

Comment:
Can you register a new type 'instance' similar to what OHIF uses, so that if a different metadata provider is used, it can safely provide a complete set of tags rather than relying on this rather adhoc collection?  Then, specifically in teh case of OHIF, it becomes possible to get a complete set.